app/lib/big_query/entity_event.rb

```diff
@@ -0,0 +1,49 @@
+module BigQuery
+  class EntityEvent
+    EVENT_TYPES = %w[create_entity update_entity].freeze
+
+    def initialize
+      @event_hash = {
+        environment: ENV["RAILS_ENV"],
```

Comment:
Just out of curiousity, is there a reason not to use `Rails.env`? Is it considered harmful in any way? Or just too new/a fad?